build.gradle.kts

```diff
@@ -42,11 +43,14 @@ if (majorVersion < MIN_JAVA_VER) {
 
     // Test on minimum supported java version
     tasks.register<Test>("testsMinJava") {
```

Comment:
If your going to copy paste the changes from #36 I would also recommend
```kt
val testsMinJava by tasks.registering(Test::class) {
```

It's more idiomatic in Gradle builds to use the named container delegates.

Reply:
I think the existing code is clearer. 

BTW, I had already fixed most of the issues locally before I saw your MR, but did get some ideas from yours.